github/resource_github_repository.go

```diff
@@ -1062,3 +1069,13 @@ func customDiffFunction(_ context.Context, diff *schema.ResourceDiff, v interfac
 	}
 	return nil
 }
+
+func updateVulnerabilityAlerts(d *schema.ResourceData, client *github.Client, ctx context.Context, owner, repoName string) error {
+	updateVulnerabilityAlerts := client.Repositories.DisableVulnerabilityAlerts
+	if vulnerabilityAlerts, ok := d.GetOk("vulnerability_alerts"); ok && vulnerabilityAlerts.(bool) {
```

Comment:
Not new behavior, but this treats the absense of `vulnerability_alerts = "true"` as an alias for `vulnerability_alerts = "false"`.

That is not true when a global org-level policy enables vulnerability alerts by default. Is this desirable behavior?

Reply:
@Lekensteyn this is a pattern used throughout the provider, the `GetOk()` function is intended to only respond if the value has been set.